tests/test-scribe.c

```diff
@@ -373,6 +387,9 @@ test_error (Fixture       *fixture,
   gboolean ret;
   g_autoptr(GError) error = NULL;
 
+  if (g_test_failed ())
```

Comment:
That’s kind of icky, but I don’t see an alternative. It’s annoying that `g_test_skip()` in the fixture setup function doesn’t actually skip the test.

Nitpick: Would be good to add a comment like `/* Did fixture setup fail? */`.

Reply:
Added. Yes, I would prefer that the test func were not run if the fixture failed/skipped but the docs suggest this is a feature (emphasis mine):

> Calling this function will not stop the test from running, you need to return from the test function yourself. So you can produce additional diagnostic messages or **even continue running the test**.